swagger.yaml

```diff
@@ -0,0 +1,493 @@
+openapi: 3.0.2
+info:
+  title: collection instrument
+  description: A service for the uploading of collection instruments and exercises
+  version: "2.0"
+
+servers:
+  - url: https://localhost:8080/
+    description: The path used by the collection exercise service when port forwarded by kubectl (8080:8080)
+
+tags:
```

Comment:
this is a little tag heavy. Get rid of all tags other than info and collection-instrument for now.

Reply:
I've reduced the tags to just `collection-instrument`, `info` and `survey-response`.